vos/vos/vos.py

```diff
@@ -1552,15 +1552,16 @@ def copy(self, source, destination, send_md5=False, disposition=False,
         :type source: unicode
         :param destination: The VOSpace location to put the file to or the
         local destination.
-        :type destination: unicode
+        :type destination: Node
         :param send_md5: Should copy send back the md5 of the destination
         file or just the size?
         :type send_md5: bool
         :param disposition: Should the filename from content disposition be
         returned instead of size or MD5?
         :type disposition: bool
-        :param fhead: Return just the headers of a FITS file
-        :type fhead: bool
+        :param head: Return just the headers of a file. Only FITS files
+        supported.
```

Comment:
The service does a file-type check as well:  It will allow the header operation if (the type is known & the type is fits) or (the type is not known).  If the type is not known and it is not a fits file the failure happens later when the actual operation is performed.